gatsby-node.js

```diff
@@ -99,4 +106,253 @@ async function makeSynoptic(createPage, reporter, graphql) {
       context: section
     })
   }
+}
+
+async function makeSearchIndex(reporter, graphql){
+  const lunr = require('lunr')
+  const remark = (await import('remark')).remark
+
+  const jsdom = require("jsdom")
+  const { JSDOM } = jsdom
+
+  const result_md = await graphql(`
+    query {
+      allMarkdownRemark {
+        edges {
```

Comment:
Unless you're planning to query node relationships (such as next and previous), skip `edges` and go straight to `nodes` like in the query below